tests/queries/0_stateless/03012_fix_age_undef_memory.sql

```diff
@@ -0,0 +1,3 @@
+SET allow_experimental_analyzer = 1;
+SELECT age('year', toDate(materialize(_CAST('2018-01-01', 'LowCardinality(String)'))), toDate('2017-01-01')) GROUP BY _CAST(17167, 'Date');
```

Comment:
If this is the reproducer, I'd suggest you to check master without changes because `GROUP BY` const column was fixed recently for the analyzer.

Changes might still be interesting / necessary anyway.

Reply:
According to the [newest test results](https://fiddle.clickhouse.com/e43a9eba-699a-4dd9-8b31-6cbee36ee134), seems that this bug is not actual now, so I will close this PR